tripal_ws/includes/TripalFields/remote__data/remote__data.inc

```diff
@@ -237,6 +237,11 @@ class remote__data extends WebServicesField {
     if (preg_match('/\?/', $query)) {
       list($path, $q) = explode('?', $query);
     }
+
+    if(empty($remote_site)) {
+      return null;
```

Comment:
I would add a tripal_report_error() before returning NULL. That way an admin would see there's a problem if looking in the recent messages log rather then left scratching their head. :-)